tests/test_api.py

```diff
@@ -0,0 +1,89 @@
+"""Tests the Bounce API."""
```

Comment:
I think you accidentally added this file. It seems to be a duplicate of `tests/test_endpoints.py`.